lib/http.js

```diff
@@ -81,17 +81,79 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     cors(),
     // FIXME: add validation
     //validate({bodySchema: updateMeterBody}),
-    _createAuthorizeMiddleware(),
+    _addMeterRecordToRequest(),
+    cors(),
+    _createAuthorizeMiddleware({
+      getExpectedTarget({req}) {
+        // only support update meter config URL as expected target; controller
+        // is the controller of the meter
+        const baseUrl = `https://${req.get('host')}${routes.basePath}`;
+        const expectedTarget =
+          `${baseUrl}/${encodeURIComponent(req.params.meterId)}`;
+        return {expectedTarget};
+      },
+      getRootController({req}) {
+        // root controller for update meter config is always the controller of
+        // the meter
+        const {meter: {controller}} = req.meterRecord;
+        return controller;
+      }
+    }),
     asyncHandler(async (req, res) => {
       let {body: meter} = req;
 
       // update meter according to handler result
       ({meter} = await HANDLERS.update({meter}));
 
-      // FIXME: update meter using meters.update()
+      // specify the max number of retries when InvalidStateError is thrown
+      const maxRetries = 10;
```

Comment:
We don't want to retry when there's a sequence error -- that can only be remedied by the client. They have to resolve the conflict. The point of raising the exception is to let them know that the config that they were basing their changes on has changed -- so they may need to fix things. If we forcibly just increment the sequence and retry it defeats the purpose of the feature (why even have a sequence?).

Reply:
That's fine. We can let the client handle the sequence error.